src/main/java/org/opendatakit/suitcase/ui/PullPanel.java

```diff
@@ -57,17 +64,29 @@ public PullPanel(IOPanel parent) {
         gbc.gridx = 0;
         gbc.gridy = GridBagConstraints.RELATIVE;
 
-        JPanel pullInputPanel = new InputPanel(
-                new String[]{"Table ID"},
-                new JTextField[]{sTableIdText},
-                new String[]{"table_id"}
+        comboBoxModel = new DefaultComboBoxModel<String>(getAllTableIds(parent.getCloudEndpointInfo()));
```

Comment:
Populating the the model here is redundant. There is a `PropertyChangeListener` already.